Source/WebCore/rendering/svg/RenderSVGText.cpp

```diff
@@ -109,34 +109,30 @@ static inline void collectLayoutAttributes(RenderObject* text, Vector<SVGTextLay
     }
 }
 
-static inline bool findPreviousAndNextAttributes(RenderElement& start, RenderSVGInlineText* locateElement, bool& stopAfterNext, SVGTextLayoutAttributes*& previous, SVGTextLayoutAttributes*& next)
+static inline bool findPreviousAndNextAttributes(RenderElement& root, RenderSVGInlineText* locateElement, SVGTextLayoutAttributes*& previous, SVGTextLayoutAttributes*& next)
 {
     ASSERT(locateElement);
-    // FIXME: Make this iterative.
-    for (CheckedRef child : childrenOfType<RenderObject>(start)) {
-        if (auto* text = dynamicDowncast<RenderSVGInlineText>(child.get())) {
+    bool stopAfterNext = false;
```

Comment:
The name `stopAfterNext` is a little bit confusing. What does `Next` mean in this context? 

I would suggest renaming it to `foundLocateElement` or something. `stopAfterNext` is set to true only when `locateElement == text` is true.